### DISCUSSION TITLE
what is default merge strategy for this repository?

### DISCUSSION THREAD

**Discussion by nijkah** [facebookresearch/detectron2#3808] [Q&A] [answered]
I wonder how FAIR manages its repositories in aspect of commit history.
I found that there was no "Merge" commit in the history.

Are all PRs merged with rebase and merge?
Is there any available problem with "rebase and merge" which has the property changes the commit id?

**Comment by nijkah** [comment, marked as answer]
I think its merge strategy is ' Squash and Merge' because every commits has a PR number.

**Reply by zhanghang1989** [reply]
Yea, it is similar to squash and merge. We actually imported the PRs to internal code base to commit.
